### ISSUE TITLE
Default Notification Body

### ISSUE THREAD

**Issue by Rithie** [robcalcroft/react-native-in-app-notification#22]
Hi, 
firstly, this lib is awesome eh. Great job!
this is not a issue per say,

I'd like to understand how can I use background props. How can I customize the notification. I couldn't find any tutorial or documentation.

**Comment by graig12** [comment]
Yes this library is awesome however how can the background Color be used

**Comment by graig12** [comment]
@Rithie  did you manage to figure this one out?

**Comment by robcalcroft** [comment]
Not sure if this is what you're looking for but you can write your own component and then give it to the library to display for you https://github.com/robcalcroft/react-native-in-app-notification#props see the `notificationBodyComponent ` prop

**Comment by graig12** [comment]
Thanks I will try. One more question 
Is it possible to use a url for the 
Icon prop in this.prop.shownotification()?

**Comment by sa2812** [comment]
@mun5865 - Once you've created your own notification component and referenced it in the `<InAppNotificationProvider notificationBodyComponent={...}>` provider, you can just change the color as you would with a simple styles component. The [example component here](https://github.com/robcalcroft/react-native-in-app-notification/tree/master/src/DefaultNotificationBody) is very useful to help you create your own
